flyteadmin/pkg/runtime/interfaces/application_configuration.go

```diff
@@ -491,8 +491,13 @@ type NotificationsProcessorConfig struct {
 type EmailServerConfig struct {
 	ServiceName string `json:"serviceName"`
 	// Only one of these should be set.
-	APIKeyEnvVar   string `json:"apiKeyEnvVar"`
-	APIKeyFilePath string `json:"apiKeyFilePath"`
+	APIKeyEnvVar           string `json:"apiKeyEnvVar"`
+	APIKeyFilePath         string `json:"apiKeyFilePath"`
+	SMTPServer             string `json:"smtpServer"`
```

Comment:
would this be cleaner in a separate object?

Reply:
When moving it out to a separate object, I would rather move out the `APIKeyEnvVar` and `APIKeyFilePath` properties into their own type. Since the name `EmailServerConfig` better matches the newly introduced properties in my eyes. Doing that would be a breaking change. Maybe you can make a proposal how you want the objects structured / named?